cmake/wayland_driver.cmake

```diff
@@ -0,0 +1,61 @@
+project(lv_wayland)
+
+option(USE_WL_SHELL "use wl shell (discard, no recommend)" OFF)
```

Comment:
I'd rephrase the comment as:

`"use wl_shell (deprecated, not recommended)"`

Reply:
modified already